public class LockerCode {

    private String combination;
    private final int lockerNumber;

    public LockerCode(int lockerNumber, String combination) {
        this.lockerNumber = lockerNumber;
        this.combination = combination;
    }

    public void changeCode(String currentCode, String newCode) {

        if (combination.equals(currentCode)) {
            combination = newCode;
            System.out.println("Code changed successfully");
        } else {
            System.out.println("Code change rejected");
        }
    }

    public static void main(String[] args) {

        LockerCode l = new LockerCode(101, "1234");

        l.changeCode("1234", "5678");
        l.changeCode("0000", "9999");
    }
}